src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -97,7 +99,7 @@ extends AbstractMap[Key, Value]
     */
   private[this] def growTable() = {
     val oldSize = mask + 1
-    val newSize = 4 * oldSize
+    val newSize = if (oldSize >= MaximumTableCapacity) Int.MaxValue - 8 else 4 * oldSize
```

Comment:
You can't do this; the mask (see 3 lines down) requires the size to be a power of two.